requirements.txt

```diff
@@ -5,3 +5,5 @@ tqdm
 more-itertools
 tiktoken
 triton>=2.0.0;platform_machine=="x86_64" and sys_platform=="linux" or sys_platform=="linux2"
```

Comment:
reinstalling torch could override the Gaudi torch already installed?

Reply:
I think it shouldn't override

Reply:
I think torch will be reinstalled: 
could you show your install and launch steps?

Reply:
@orionsBeltWest 
I just added the [Dockerfile_hpu](https://github.com/BlueLabelLabs/whisper_fork/pull/2/files#diff-28de834099a6f1c52d83e6dff545f6c282de474af492ae17f8875437373c10d9)  
And info about [the new section in README.md](https://github.com/BlueLabelLabs/whisper_fork/pull/2/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5)  

In the Dockerfile_hpu I added installing packages just with adding:  
```txt
# Install Python packages from Whisper requirements
RUN pip install --upgrade pip \
    && pip install -r requirements.txt \
```
and the requirements.txt are:  
```txt
numba
numpy
torch
tqdm
more-itertools
tiktoken
triton>=2.0.0;platform_machine=="x86_64" and sys_platform=="linux" or sys_platform=="linux2"
scipy
pytest
```

Reply:
Take a look at vllm repo It has a good example of specifying various requirements.txt files and Dockerfile for various accelerators. https://github.com/vllm-project/vllm

Reply:
Thank you for providing this repository. I have also adjusted the current Dockerfile to integrate with Whisper, including the installation of `ffmpeg` - is is mentioned in README.md by Whisper. It seems that the current version is now correct. What do you think about it?